adapters/http-base/src/main/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapter.java

```diff
@@ -296,12 +288,6 @@ protected Router createRouter() {
             }
             ctx.next();
         });
-
-        // 4. BodyHandler with request size limit
```

Comment:
Concerning the removal of the BodyHandler here:
Besides the `VertxBasedHttpProtocolAdapter.addRoutes` method, there are also `addRoutes` implementations in `LoraProtocolAdapter` and `SigfoxProtocolAdapter`.  
The BodyHandler should be added there as well (on routes that handle the upload of a telemetry/event message).